.github/pnnl-ci/README.md

```diff
@@ -97,6 +80,49 @@ Since the pipeline status is automatically configured through GitLab premium + G
 
 There is a way to orchestrate this pipeline posting through non-premium GitLab as well - https://ecp-ci.gitlab.io/docs/guides/build-status-gitlab.html...
 
+#### Personal Access Token
+
+A Personal Access Token is needed to enable GitHub/GitLab integration.
+
+@jaelynlitz currently holds the PAT, set to expire Feb 2025. Followed instructions [here](https://docs.gitlab.com/ee/ci/ci_cd_for_external_repos/github_integration.html) under "Connect manually".
+
+In Summary:
+1. in [GitHub](https://github.com/settings/tokens/new), generate a Personal Access Token (classic) with permissions `repo` and `admin:repo_hook`
+2. have Owner permissions in GitLab
+3. in GitLab, go to Settings > Integrations > GitHub - paste your PAT in the new token field, test settings to see if connection is successful, then save changes.
+
+### Secrets
+
+The GitHub action in `/.github/workflows/pnnl_push_mirror.yml` relies on the following GitHub secrets. Make sure to configure these if they are expired/broken:
```

Comment:
Worth linking to GitHub documentation on secrets?

Reply:
Linked 😄